themes/wds_headless/inc/acf-pro.php

```diff
@@ -67,6 +67,74 @@ function wds_acf_blocks_init() {
 				'supports'        => $supports,
 			]
 		);
+
+		// Banners.
+		acf_register_block_type(
+			[
+				'name'            => 'banners',
+				'title'           => esc_html__( 'Banner', 'wds' ),
+				'description'     => esc_html__( 'A block to display a banners post.', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
+				'icon'            => 'images-alt2',
+				'keywords'        => [ 'banners' ],
+				'mode'            => 'edit',
+				'enqueue_assets'  => '',
+				'align'           => 'wide',
+				'supports'        => $supports,
+			]
+		);
+
+		// Explore block.
+		acf_register_block_type(
+			[
+				'name'            => 'explore-list',
+				'title'           => esc_html__( 'Explore List', 'wds' ),
+				'description'     => esc_html__( 'A block to display list of explore cards.', 'wds' ),
+				'render_callback' => '',
+				'category'        => 'wds-content',
```

Comment:
can we customize the category?

Reply:
@KobbyMmo here, we can pass any string. where this block will be grouped to in blocks list? So its possible.

Reply:
I have replaced that with `Section`